src/components/Map/index.tsx

```diff
@@ -39,22 +20,22 @@ import {
   ZINDEX_MARKER_MOUSE_OUT,
   ROUTE,
   SIDO_NAMES,
+  INIT_DUST_INFO,
+  INIT_DUST_SCALE,
+  INIT_DUST_GRADE,
+  INIT_STALE_TIME,
 } from '@/utils/constants';
+import { makeCityMarker, makeSidoMarker, removeMarker } from '@/utils/markers';
 import ControlButton from './ControlButton';
 
 const Map = () => {
   const mapRef = useRef<HTMLDivElement | null>(null);
-  const sidoDustInfoMarkers: kakao.maps.CustomOverlay[] = [];
   const [cityDustInfoMarkers, setCityDustInfoMarkers] = useState<
     kakao.maps.CustomOverlay[]
   >([]);
-  const [city, setCity] = useState('동네 정보를 받아오지 못했어요');
-  const [dustInfo, setDustInfo] = useState({
-    fineDustScale: 0,
-    fineDustGrade: 0,
-    ultraFineDustScale: 0,
-    ultraFineDustGrade: 0,
-  });
+  const [selectedSidoOrCity, setSelectedSidoOrCity] =
+    useState('동네 정보를 받아오지 못했어요');
```

Comment:
```suggestion
  const [selectedSidoOrCity, setSelectedSidoOrCity] = useState(NO_CITY_DATA);
```
만들어둔 상수 적용하면 좋을 것 가타용